src/plugins/intel_gpu/src/plugin/transformations/transpose_fusion.cpp

```diff
@@ -162,14 +162,7 @@ TransposeMatMulMatcher::TransposeMatMulMatcher() {
         return std::dynamic_pointer_cast<ov::op::v1::Transpose>(output.get_node_shared_ptr()) == nullptr
                && is_fp_type(output);
     };
-    auto is_dynamic = [](const ov::Output<ov::Node>& output) -> bool {
-        bool is_dynamic = output.get_node_shared_ptr()->get_output_partial_shape(0).is_dynamic();
-        size_t num_inputs = output.get_node_shared_ptr()->get_input_size();
-        for (size_t idx = 0; idx < num_inputs; idx++) {
-            is_dynamic |= output.get_node_shared_ptr()->get_input_partial_shape(idx).is_dynamic();
-        }
-        return is_dynamic;
-    };
+
```

Comment:
As I remember there was perf issue on static models (stable diffusion I think) with gemm fused with transpose. https://github.com/openvinotoolkit/openvino/pull/22826
@e-ddykim  Is the issue gone after sdpa is being used?

Reply:
Sorry for too late reply. Anyway, the issue is gone after SDPA is enabled.